app/build.gradle

```diff
@@ -40,4 +45,10 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+
+    //viewModel
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
do you really need this implementation ? if answer is no, delete it

Reply:
done